sync/syncer.go

```diff
@@ -433,43 +442,54 @@ func validateUniqueTxAtx(b *types.Block) error {
 	return nil
 }
 
-func (s *Syncer) BlockSyntacticValidation(block *types.Block) ([]*types.AddressableSignedTransaction, []*types.ActivationTx, error) {
-	// run block fast validation
-	if err := s.blockFastValidator(block); err != nil {
-		s.With().Error("BlockSyntacticValidation: block validation failed", log.BlockId(uint64(block.ID())), log.Err(err))
+func (s *Syncer) blockSyntacticValidation(block *types.Block) ([]*types.AddressableSignedTransaction, []*types.ActivationTx, error) {
```

Comment:
Are you rebased on develop? Where is the call to fastValidation?

Reply:
it happens inside the validation queue for blocks that are fetched
and in the block_listener for blocks that are gossiped